### ISSUE TITLE
[A11y]C ontrasts partly seem not to be WCAG-compliant (AAA & AA)

### ISSUE THREAD

**Issue by mariohamann** [corona-warn-app/cwa-app-android#174]
## Current Implementation
Ich habe die [High-Resolution-Screenshots](https://github.com/corona-warn-app/cwa-documentation/blob/master/ui_screens.md) mit dem [WebAIM Contrast-Checker](https://webaim.org/resources/contrastchecker/) analyisert. Dabei zeigte sich, dass einzelne Kontraste WCAG AAA nicht erreichen, mitunter nicht einmal WCAG AA.

(The following image may show other colours because of web compression.)
![contrasts](https://user-images.githubusercontent.com/26542182/83799430-12e4fc00-a6a6-11ea-996e-290ae74b17b3.jpg)

Screen 1 (Startscreen): Sowohl die grüne Fläche als auch der blaue Button bieten nicht genug Kontraste. Mit dem Alter lässt außerdem die Wahrnehmung der Farbe "Blau" obendrein am stärksten nach ([Quelle](https://www.smashingmagazine.com/2015/02/designing-digital-technology-for-the-elderly/)) weswegen der untere Button in Kombination mit der durchgehenden Großschreibung und dem schmalen Schriftschnitt für bestimmte Nutzergruppen Probleme bereiten könnte. 

Screen 2 ("Niedriges Risiko") und 3 ("Risiko-Ermittlung") beinhalten beide eine kleine graue Schrift. Diese ist aus Kontrastperspektive gerade noch über der AA Mindestgrenze von 4.5, könnte jedoch in Kombination mit der sehr kleinen Schrift für manche Nutzergruppen sehr schlecht zu lesen sein. 

Screen 3 ("Risikoermittlung"): Der Switch könnte für manche Nutzergruppen nicht als solcher erkennbar sein – manche könnten einfach nur einen Punkt sehen.
## Suggested Enhancement
Aus Gründen der Barrierefreheit sollten die Kontraste erhöht werden. Gleichzeitig muss natürlich auch der Unterschied zwischen Überschriften und "Nebentext" gewahrt werden. Es könnte sich anbieten, hier eine stärkere Unterscheidung durch entsprechende Schriftschnitte zu schaffen.

## Expected Benefits
Die Applikation sollte für alle Nutzergruppen gut bedienbar sein – einzelne Risikogruppen wie z. B. Seniorinnen haben aber mit hoher Wahrscheinlichkeit erhöhte Anforderungen an die Usability auch in Form der Kontraste. Um diese Nutzergruppen nicht von der Nutzung auszuschließen, sollte WCAG AAA bestmöglich umgesetzt werden.I found that some of the contrasts do not reach WCAG AAA, sometimes not even WCAG AA.

(The following image may show other colours because of web compression.)
![contrasts](https://user-images.githubusercontent.com/26542182/83799430-12e4fc00-a6a6-11ea-996e-290ae74b17b3.jpg)

Screen 1 (start screen): Both the green area and the blue button do not offer enough contrast. In addition, the perception of the color "blue" decreases most with age ([Source](https://www.smashingmagazine.com/2015/02/designing-digital-technology-for-the-elderly/)), which is why the lower button in combination with the continuous capitalization and the narrow font could cause problems for certain user groups. 

Screen 2 ("Low risk") and 3 ("Risk determination") both contain a small grey font. This is just above the AA minimum of 4.5 from a contrast perspective, but in combination with the very small font could be very difficult to read for some user groups. 

Screen 3 ("Risk assessment"): The switch may not be recognizable as such for some user groups - some may just see a dot.
## Suggested Enhancement
For reasons of accessibility, the contrasts should be increased. At the same time, the difference between headings and "secondary text" must of course be maintained. It might be advisable to create a stronger distinction here by using appropriate font styles.

## Expected Benefits
The application should be easy to use for all user groups - but individual risk groups such as senior citizens are very likely to have higher usability requirements, also in terms of contrasts. In order not to exclude these user groups from using the application, WCAG AAA should be implemented in the best possible way.

**Comment by tkowark** [comment]
Dear @mariohamann , thanks for pointing us to that issue. I'll move it to the documentation repository and forward it to the design team. They will then be able to answer and provide potential solutions. 

I'll further close the identical issue in the iOS repo, since they will have to be answered together, anyway.
